arcgis-ios-sdk-samples/Utility network/Trace utility network/TraceUtilityNetworkViewController.swift

```diff
@@ -27,38 +27,9 @@ class TraceUtilityNetworkViewController: UIViewController, AGSGeoViewTouchDelega
     
     private let featureServiceURL = URL(string: "https://sampleserver7.arcgisonline.com/arcgis/rest/services/UtilityNetwork/NapervilleElectric/FeatureServer")!
     
-    // Create electrical distribution line layer ./115 and electrical device layer ./100.
-    private var layers: [AGSFeatureLayer] {
-        return [115, 100].map {
-            let featureTable = AGSServiceFeatureTable(url: featureServiceURL.appendingPathComponent("\($0)"))
-            let layer = AGSFeatureLayer(featureTable: featureTable)
-            if $0 == 115 {
-                // Define a solid line for medium voltage lines and a dashed line for low voltage lines.
-                let darkCyan = UIColor(red: 0, green: 0.55, blue: 0.55, alpha: 1)
-                let mediumVoltageValue = AGSUniqueValue(
-                    description: "N/A",
-                    label: "Medium voltage",
-                    symbol: AGSSimpleLineSymbol(style: .solid, color: darkCyan, width: 3),
-                    values: [5]
-                )
-                let lowVoltageValue = AGSUniqueValue(
-                    description: "N/A",
-                    label: "Low voltage",
-                    symbol: AGSSimpleLineSymbol(style: .dash, color: darkCyan, width: 3),
-                    values: [3]
-                )
-                layer.renderer = AGSUniqueValueRenderer(
-                    fieldNames: ["ASSETGROUP"],
-                    uniqueValues: [mediumVoltageValue, lowVoltageValue],
-                    defaultLabel: "",
-                    defaultSymbol: AGSSimpleLineSymbol()
-                )
-            }
-            return layer
-        }
-    }
-    
+    private var layers = [AGSFeatureLayer]()
```

Comment:
It looks like this property isn't even needed as it is only used in `init(coder:)`.